prometheus_client/metrics.py

```diff
@@ -706,6 +706,8 @@ def info(self, val: Dict[str, str]) -> None:
             raise ValueError('Overlapping labels for Info metric, metric: {} child: {}'.format(
                 self._labelnames, val))
         with self._lock:
+            if any(i is None for i in val.values()):
+                raise ValueError('Label value cannot be None')
```

Comment:
One other thing, there's no need to have the lock while doing this check, could you move this logic up a line?

Apologies, I should have seen this in the first review.